tensorflow/tools/ci_build/linux/rocm/run_csb_tests.sh

```diff
@@ -0,0 +1,56 @@
+#!/usr/bin/env bash
+# Copyright 2017 The TensorFlow Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# ==============================================================================
+
+set -e
+set -x
+
+N_JOBS=$(grep -c ^processor /proc/cpuinfo)
+N_GPUS=$(lspci|grep 'VGA'|grep 'AMD/ATI'|wc -l)
+
+echo ""
+echo "Bazel will use ${N_JOBS} concurrent build job(s) and ${N_GPUS} concurrent test job(s)."
+echo ""
+
+# Run configure.
+export PYTHON_BIN_PATH=`which python3`
+export CC_OPT_FLAGS='-mavx'
+
+export TF_NEED_ROCM=1
+export TF_GPU_COUNT=${N_GPUS}
+
+yes "" | $PYTHON_BIN_PATH configure.py
```

Comment:
JFYI, this is not testing XLA at all.
in case you also want to test XLA.

Reply:
>JFYI, this is not testing XLA at all.

that is intentional. ROCm support for XLA is still in the process of being upstreamed. 

XLA unit tests will not pass until the upstreaming of ROCm support is complete, and hence leaving those out for now. Will enable them once the upstreaming process is complete.